### DISCUSSION TITLE
Plugin Idea : Playnite Plugin

### DISCUSSION THREAD

**Discussion by lenicyl** [Flow-Launcher/Flow.Launcher#966] [Ideas]
[Playnite](https://playnite.link) is a great software for managing a game library across multiple platforms

It would be really cool to have a plugin to launch games from flow, similar to the epic games plugin.

Playnite already has a [similar plugin](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwjcwNSGs6L1AhUrG6YKHQ1hAaMQFnoECAQQAQ&url=https%3A%2F%2Fplaynite.link%2Fforum%2Fthread-677-post-4789.html&usg=AOvVaw2516-n_-4zl2wyQqrfW8bT) but thats for use inside playnite, it would be sick if it were there on flow :D 

Thanks ~

**Comment by Garulf** [comment]
Hi, I'm the author of both the Steam and Epic Games plugin. Playnite was brought to my attention by fellow Team member @onesounds and is in a planning/research stage.

**Reply by lenicyl** [reply]
Interesting, looking forward to that
Good luck :0

**Comment by Garulf** [comment]
Sadly Playnite doesn't seem to expose anyway to access your game library *outside* of Playnite. Only internal extensions/plugins can access your library.

**Reply by lenicyl** [reply]
:0 kinda sucks,
Maybe i could check the game list explorer plugin later today to maybe fork it to be usable if its in powershell ?
![image](https://user-images.githubusercontent.com/73347143/148670806-be6758e3-9ed4-441b-8760-919e2dfdf68f.png)

Edit : yep its written in powershell, [here is the link ](https://github.com/JosefNemec/PlayniteExtensions/blob/master/source/Generic/LibraryExporter/LibraryExporter.psm1)

**Reply by Garulf** [reply]
Oh yeah, I already explored this option... It's not ideal but it works:
![image](https://user-images.githubusercontent.com/535299/148682740-695a75a1-117c-4b53-93ad-e1cfbc8aae33.png)

Just need to clean up a few details and it should be released soon.

**Comment by Garulf** [comment]
Plugin has been submitted: https://github.com/Flow-Launcher/Flow.Launcher.PluginsManifest/pull/123

Plugin's page can be found here: https://github.com/Garulf/Playnite-Plugin

Playnite plugin was submitted here: https://github.com/JosefNemec/PlayniteAddonDatabase/pull/103

The plugin page can be found here: https://github.com/Garulf/FlowLauncherExporter

Just need to wait for these submissions to be accepted on their respective repositories.
